### PR TITLE
Base snippet

### PR DESCRIPTION
**Pull request by sylwiabr**
### Pull Request Description

Improve the code snippets for standard library
### Checklist

- [x] The documentation has been updated if necessary.
- [x] The code conforms to the [Luna Style Guide](https://github.com/luna/wiki/blob/master/code-style/01.general.md) if it is Haskell.
- [x] The code has been tested where possible.

### CODE REVIEW

File: env/stdlib/Std/snippets.yaml [hunk 1]
```diff
@@ -1,5 +1,207 @@
-Std.HTTP:
+Std.Base:
   $global:
+    - name: pi
+      documentation: The mathematical constant π = 3.14159265359.
+    - name: Map.singleton "foo" 10
+      documentation: Creates a map with a single element.
```
Comment (iamrecursion) [inline env/stdlib/Std/snippets.yaml]: I would say what the key and value are. Just to make it more explicit.

File: env/stdlib/Std/snippets.yaml [hunk 2]
```diff
@@ -1,5 +1,207 @@
...
+    - name: JSON.empty
+      documentation: Empty JSON object.
+    - name: readFile "/example/path/to/data"
```
Comment (iamrecursion) [inline env/stdlib/Std/snippets.yaml]: Can we give them a snippet to a known good file instead? I'm not sure what the point of one is that is just a dummy path.

File: env/stdlib/Std/snippets.yaml [hunk 3]
```diff
@@ -1,5 +1,207 @@
...
+      documentation: Read file from the hard drive.
+    - name: writeFile "/example/path/to/file" "lorem ipsum"
```
Comment (iamrecursion) [inline env/stdlib/Std/snippets.yaml]: Like above, a propely writeable file might be better, especially given these are likely going to execute on load into LS.

File: env/stdlib/Std/snippets.yaml [hunk 4]
```diff
@@ -1,5 +1,207 @@
...
+      documentation: TimeInterval in minutes.
+    - name: hours
+      documentation: TimeInterval in hours.
+    - name: days
+      documentation: TimeInterval in days.
+    - name: weeks
+      documentation: TimeInterval in weeks.
+  Real:
+    - name: + 3.0
+      documentation: Add two floats.
```
Comment (iamrecursion) [inline env/stdlib/Std/snippets.yaml]: We call it `Real`, so do we want the snippets to refer to them as 'floats'?

File: env/stdlib/Std/snippets.yaml [hunk 5]
```diff
@@ -1,5 +1,207 @@
...
+      documentation: Return list of all characters in the text.
+    - name: lowercase
+      documentation: Change all characters to lowercase.
+    - name: reverse
+      documentation: Reverse the text.
+    - name: uppercase
+      documentation: Change all characters to uppercase.
+  Complex:
+    - name: modulus
+      documentation: Calculate modulus.
```
Comment (iamrecursion) [inline env/stdlib/Std/snippets.yaml]:
```suggestion
      documentation: Calculate complex modulus.
```

While this is clear in many contexts, there are some where it wouldn't be without the addition.

File: env/stdlib/Std/snippets.yaml [hunk 6]
```diff
@@ -1,5 +1,207 @@
...
+  Stream:
+    - name: drop 1
+      documentation: Drop the given number of elements from the stream.
+    - name: take 1
+      documentation: Return the prefix of the stream of given length.
+    - name: consume 5
+      documentation: List of the first `n` elements of stream and the result of dropping the first `n` values of it.
+
+Std.Geo:
+  GeoJSONFeature:
```
Comment (iamrecursion) [inline env/stdlib/Std/snippets.yaml]: Would be nice to have a few more for this.

File: env/stdlib/Std/snippets.yaml [hunk 7]
```diff
@@ -1,5 +1,207 @@
...
+    - name: addProperty "popupContent" "home"
+      documentation: Add a property do GeoJSONFeature.
+
+Std.HTTP:
+  $global:
```
Comment (iamrecursion) [inline env/stdlib/Std/snippets.yaml]: And here too.

File: env/stdlib/Std/snippets.yaml (hunk 2 above)
Comment (sylwiabr) [reply env/stdlib/Std/snippets.yaml]: We will as soon as https://github.com/luna/luna/pull/394 will be merged

File: env/stdlib/Std/snippets.yaml (hunk 6 above)
Comment (sylwiabr) [reply env/stdlib/Std/snippets.yaml]: then we need to improve the library :)

File: env/stdlib/Std/snippets.yaml (hunk 2 above)
Comment (iamrecursion) [reply env/stdlib/Std/snippets.yaml]: Brilliant. If you want to merge this ASAP, then maybe file an issue to track that improvement!

Comment (sylwiabr) [reply env/stdlib/Std/snippets.yaml]: https://github.com/luna/luna-studio/issues/1366

**Comment by sylwiabr** [comment]
It will come to Luna studio with Luna bump

**Pull request closed without merging**